templates/web.yml.template

```diff
@@ -166,6 +166,8 @@ uiConfig:
   bdCatalystTerraImportUrl: {{ BD_CATALYST_TERRA_IMPORT_URL }}
   bdCatalystSevenBridgesImportUrl: {{ BD_CATALYST_SEVEN_BRIDGES_IMPORT_URL }}
   elwaziImportUrl:  {{ ELWAZI_IMPORT_URL }}
+  colabImportUrl: {{ COLAB_IMPORT_URL }}
+  mybinderImportUrl: {{ MYBINDER_IMPORT_URL }}
```

Comment:
Should this me `myBinderImportUrl`?

Reply:
Just to make sure we're all on the same page, the source term is "MyBinder", which I'll call the "org part" of the `ImportUrl`.  We've got both the camelcased and lowercase org parts throughout.  Camelcased generally is more predominant.  In this case, I considered both, but went with the convention of the last [adjacent] change (`elwaziImportUrl` from "eLwazi"), because it was locally consistent and I thought it was easier to read.

Can change but I think I'll need to tweak the webservice also, that PR has merged.